lib/checks/mobile/meta-viewport-scale.js

```diff
@@ -26,10 +26,12 @@ if (
 }
 
 if (!lowerBound && result['user-scalable'] === 'no') {
+	this.data(['user-scalable=no']);
 	return false;
 }
 
 if (result['maximum-scale'] && parseFloat(result['maximum-scale']) < minimum) {
+	this.data(['maximum-scale']);
```

Comment:
Are we saying if `user-scalable=no` is already set on data, `maximum-scale` will then override the data? Should we not concat?

Reply:
It returns false. There can only be one result.